scripts/install_third_party_libs.py

```diff
@@ -305,6 +371,9 @@ def main() -> None:
     # Compile protobuf files.
     print('Installing buf and protoc binary.')
     install_buf_and_protoc()
+    print('Updating android proto files.')
```

Comment:
I'm not sure I'd call these "android proto files". This is the web codebase and it only interacts with oppia-proto-api, not oppia-android directly.

Maybe refer to them as the files from oppia-proto-api or similar.

Reply:
Fixed